README.md

```diff
@@ -342,6 +342,18 @@ Note: This is same as `getId();` the difference being it only returns the first
 const requestIdentifier = store.getShortId();
 ```
 
+### reset()
+
+Reset the store or a specific key of the global store.
+
+- `@returns {void}`
+
+```js
+store.reset(); // Reset the whole store
+
+store.reset('foo') // It will delete the key foo from store and get will result undefined
+```
```

Comment:
@samirsilwal I believe this is autogenerated, no need to add it yourself. It is extracted from the doc block. Please run `yarn doc:update`.

Reply:
cool

Reply:
I don't see the command in package.json

Reply:
Oops I confused it with sync-db. GG. Your change is good. :)